### ISSUE TITLE
codecept def looks for default config path

### ISSUE THREAD

**Issue by masterjg** [Codeception/CodeceptJS#879]
#### What are you trying to achieve?
I am trying to generate definitions for project with custom config path. `codeceptjs def` however doesn't have `--config` option.

#### What do you get instead?
```
Can not load config from codecept.conf.js or codecept.json
CodeceptJS is not initialized in this dir. Execute 'codeceptjs init' to start
```

### Details
* CodeceptJS version: 1.1.1

**Comment by DavertMik** [comment]
Yes, it's not aware of config option. It looks like an easy pick. Could you implement that feature and send a pull request?

**Comment by reubenmiller** [comment]
Fixed in v1.1.6 which was release today.

You can now use the command `codeceptjs def --config ./myconfig/path/config.json`
